src/kaleidoscope/plugin/TapDance.h

```diff
@@ -50,11 +50,16 @@ class TapDance : public kaleidoscope::Plugin {
   EventHandlerResult afterEachCycle();
 
  private:
+  static constexpr uint8_t TAPDANCE_KEY_COUNT = 16;
+  typedef struct {
```

Comment:
Since this is C++, not C, the typedef is unnecessary. I find it much less confusing if it's:
```c++
struct TapDanceState { … };
```